lib/galaxy/model/migrate/versions/util.py

```diff
@@ -135,10 +144,14 @@ def add_index(index_name, table, column_name, metadata=None, **kwds):
     :param metadata: Needed only if ``table`` is a table name
     :type metadata: :class:`Metadata`
     """
+    if len(index_name) > 63 and metadata.bind.name in ('postgres', 'postgresql'):
```

Comment:
```suggestion
    if len(index_name) > MAX_INDEX_LEN and metadata.bind.name in ('postgres', 'postgresql'):
```
Define at top of file; cite https://www.postgresql.org/docs/12/runtime-config-preset.html#GUC-MAX-IDENTIFIER-LENGTH
So that.. no magic numbers? :-)